src/osd/OSDMap.h

```diff
@@ -378,7 +378,7 @@ class OSDMap {
     mempool::osdmap::vector<std::string> old_erasure_code_profiles;
     mempool::osdmap::map<int32_t,entity_addrvec_t> new_up_client;
     mempool::osdmap::map<int32_t,entity_addrvec_t> new_up_cluster;
-    mempool::osdmap::map<int32_t,uint32_t> new_state;             // XORed onto previous state.
+    mempool::osdmap::map<int32_t,uint32_t> new_state;             // ORed onto previous state.
```

Comment:
see https://github.com/ceph/ceph/blob/912f6fab28d18704f9cca05c9537c231a5462c83/src/osd/OSDMap.cc#L2165

so i think "XOR'ed" is accurate and correct.

Reply:
I'll abandon this commit.